dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -1138,6 +1138,36 @@
 				STM32_DMA_16BITS) 0>;
 			status = "disabled";
 		};
+
+		comp1_ch1: comparator@5800380c {
+			compatible = "st,stm32-comp", "st,stm32h7-comp";
+			reg = <0x5800380c 0x200>;
+			interrupts = <137 0>;
+			st,exti-line = <20>;
+			st,blank-sel = "NONE";
+			st,power-mode = "MEDIUMSPEED";
+			hysteresis = "NONE";
+			positive-input = "IN0";
+			negative-input = "VREFINT";
+			invert-output = "NONINVERTED";
```

Comment:
These should be defined by the user, best leave them undefined to force the user to configure it fully.

Reply:
done. Very good advice